src/lib/components/AnnouncementBanner/AnnouncementBanner.scss

```diff
@@ -0,0 +1,35 @@
+.usa-banner.ldaf-banner--announcement {
+  background: $primary-darkest;
+
+  .usa-banner__header-text,
+  .usa-banner__guidance {
+    color: white;
```

Comment:
nit: I know this isn't necessary, but we should probably use a variable here for consistency's sake.